torch/csrc/jit/passes/quantization.cpp

```diff
@@ -611,23 +614,21 @@ class InsertQuantDeQuantHelper {
   void run(
     script::Module& module,
     const std::string& method_name);
-  // quantization parameters and scalar type
-  std::tuple<IValue, IValue> getQParams(script::Module& module, Value* v);
+  // quantization parameters including scale, zero_point,
+  // scalar_type and axis(for per channel quantization)
+  QParamMap getQParamMap(script::Module& module, Value* v);
```

Comment:
The comment describes what q-param is, not what the function does. The description of qparams should be probably moved to `using QParamMap = ...` line.